dotCMS/src/main/java/com/dotcms/aspects/aspectj/ContextCloseableAspect.java

```diff
@@ -0,0 +1,44 @@
+package com.dotcms.aspects.aspectj;
+
+import com.dotcms.aspects.MethodInterceptor;
+import com.dotcms.aspects.interceptors.ContextCloseableMethodInterceptor;
+import com.dotcms.business.LocalTransactional;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.business.LocalTransactional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)